spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/RddSelectPdTrf.scala

```diff
@@ -0,0 +1,130 @@
+package org.apache.spark.sql.cassandra
+
+import com.datastax.spark.connector.NamedColumnRef
+import com.datastax.spark.connector.rdd.CassandraRDD
+import org.apache.spark.sql.catalyst.expressions._
+import org.apache.spark.sql.sources.Filter
+import org.apache.spark.sql.sources
+import org.apache.spark.sql.types.DataType
```

Comment:
3rd party imports should go above datastax.
datastax external (like java driver) should go above ours